OOP Fundamentals and C#/09 - CreationalPatterns/Factory/FactoryMethod/ITransport.cs

```diff
@@ -4,6 +4,9 @@ public interface ITransport
     {
```

Comment:
It's better to use abstract class here, because we force child classes to have the same properties. But they don't have any logic there